### PR TITLE
Add wallet connection

### PR DESCRIPTION
**Pull request by lserra-iov**
- Require connection for peg-in and peg-out
- Show connected account on topbar
- Show dialog if connection fails

### CODE REVIEW

File: src/common/store/session/actions.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ export const actions: ActionTree<SessionState, RootState> = {
     };
     const rLogin = state.rLoginInstance === undefined ? new RLogin({
       cacheProvider: false,
+      defaultTheme: 'dark',
```
Comment (alexjavabraz) [inline src/common/store/session/actions.ts:53]: The defaultTheme implies in every time use dark mode?

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/common/store/session/actions.ts (hunk 1 above)
Comment (lserra-iov) [reply src/common/store/session/actions.ts:53]: Yes, unless user change it in rlogin popup. We need to find a way to sync this theme and app's theme.

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**